test/src/androidTest/java/co/condorlabs/customcomponents/test/EditTextEmailFieldTest.kt

```diff
@@ -85,10 +86,12 @@ class EditTextEmailFieldTest : MockActivityTest() {
         val text = "kdfkd"
         val txtInputLayout = (ruleActivity.activity.findViewById<View>(R.id.tlEmail) as? EditTextEmailField)
         txtInputLayout?.setIsRequired(true)
+        val formField = ruleActivity.activity.findViewById<EditTextEmailField>(R.id.tlEmail)
+        val view = Espresso.onView(ViewMatchers.withId(formField!!.editText!!.id))
 
         // When
         txtInputLayout?.setRegex(android.util.Patterns.EMAIL_ADDRESS.toString())
-        Espresso.onView(ViewMatchers.withId(R.id.etEmail)).perform(typeText(text))
+        view.perform(typeText(text))
```

Comment:
These tests are no testing if the error is shown. It just tests if the error if produced.

Reply:
the update on these test its just for the diatomic ID I don't touch the original Idea of the test